pkgs/development/libraries/libusb1/default.nix

```diff
@@ -31,12 +33,23 @@ stdenv.mkDerivation rec {
 
   dontDisableStatic = withStatic;
 
-  configureFlags = lib.optional (!enableUdev) "--disable-udev";
+  configureFlags =
+    lib.optional (enableUdev) "--disable-udev"
+    ++ lib.optional (withExamples) "--enable-examples-build";
 
   preFixup = lib.optionalString enableUdev ''
     sed 's,-ludev,-L${lib.getLib udev}/lib -ludev,' -i $out/lib/libusb-1.0.la
   '';
 
+  preInstall = lib.optionalString withExamples ''
+    mkdir -p $out/{bin,sbin,examples/bin}
```

Comment:
Does this really need to be in `preInstall`? Maybe it's enough to have this line as the first line of `postInstall`?

Reply:
i'll move to postInstall